docs/organizations/overview.mdx

```diff
@@ -5,26 +5,140 @@ description: Learn about how to create and manage Clerk organizations and their
+
+## Organization invitations
+
+Organization invitations are a way to invite users to join an organization. When a user is invited to an organization, they will receive an email with a link to accept the invitation. Once they accept the invitation, they will be added as a member of the organization.
+
+By default, only admins can invite users to an organization.
+
+This feature requires that **Email address** is enabled as an [identifier](/docs/authentication/configuration/sign-up-sign-in-options#identifiers), as Clerk uses the user's email address to send the invitation. You can still disable **Email address** as an authentication option if you do not want users to be able to sign-in with their email address.
+
+To configure your application's **Email address** settings:
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username).
+2. In the navigation sidebar, select **User & Authentication > Email, Phone, Username**.
+3. In the **Contact information** section, ensure that **Email address** is toggled on.
+4. Next to **Email address**, select the settings cog icon to configure the email address settings. Here, at least **Require** should be toggled on.
+
+## Organization management
+
+As the application owner, you have control over all of the organizations within your application - both those created by you and those created by your users. You can create, update, and delete organizations, as well as manage their members and settings.
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations).
+2. In the navigation sidebar, select **Organizations**. Here, you can view and manage all organizations within your application.
+3. Select a specific organization to view its details, members, and settings. Here, you can update the organization's name, slug, and logo. You can also set the organization's [membership limit](#how-many-organizations-can-i-create) and public and private metadata.
+
+For managing organizations in your application, Clerk provides a set of prebuilt components:
+
+- [`<CreateOrganization />`](/docs/components/organization/create-organization) - A form for a user to create a new organization.
+- [`<OrganizationProfile />`](/docs/components/organization/organization-profile) - A profile page for the user's currently active organization. Out of the box, this component's **General** tab displays the organization's information and the **Leave organization** button. Admins will be able to see the **Update profile** button, **Verified domains** section, and **Delete organization** button. The **Members** tab shows the organization's members along with their join dates and roles. Admins will have the ability to invite a member, change a member's role, or remove them from the organization. Admins will have tabs within the **Members** tab to view the organization's invitations and requests.
```

Comment:
```suggestion
- [`<OrganizationProfile />`](/docs/components/organization/organization-profile) - A profile page for the user's currently active organization.
```

I know this is an aggressive cut, but IMO we should leave the detailed info about these components in their respective docs or guides, and should just link to them from an overview. I think an overview shouldn't get too detailed, and having this information here instead of somewhere more specific, or both here _and_ somewhere more specific, is a little more confusing an experience for users.

What do you think?

Reply:
So could we move this to the introductory paragraph of the [component reference](https://clerk.com/docs/components/organization/organization-profile)? Because it needs to be thoroughly laid out to users how the component works out of the box, so they can understand its purpose and what it can be used for before implementing it into their app. Also helps create awareness of all the features that are available!

Reply:
I personally think that would be better yes because it creates a central location for this info that we can link to from everywhere